Docs for resource-based sampling: remote and local instruction setting (#23466)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_mechanisms.md

#### Remote configuration

> Sampling rate configuration in the Agent is configurable remotely if you are using Agent version [7.42.0][20] or higher. Get started with [Remote Configuration][23] and start setting this parameter from the [Ingestion Control page][5]. Remote Configuration allows you to change the parameter without having to restart the Agent. The remotely set configuration takes precedence over local configurations - environment variables and `datadog.yaml` setting.

Reviewer: ```suggestion
Sampling rate configuration in the Agent is configurable remotely if you are using Agent version [7.42.0][20] or higher. To get started, set up [Remote Configuration][23] and then configure the `ingestion_reason` parameter from the [Ingestion Control page][5]. Remote Configuration allows you to change the parameter without having to restart the Agent. Remotely set configuration takes precedence over local configurations, including environment variables and settings from `datadog.yaml`.
```

- Set a **rate limit** on the number of ingested traces per second. The default rate limit is 100 traces per second per service instance (when using the Agent [default mechanism](#in-the-agent), the rate limiter is ignored).

**Note**: Sampling rules are also head-based sampling controls. If the traffic for a service is higher than the configured maximum traces per second, then traces are dropped at the root. It does not create incomplete traces.

The configuration can be set by environment variables or directly in the code:

{{< tabs >}}
{{% tab "Java" %}}
**Remote configuration**

<div class="alert alert-info"><strong>Resource-based sampling rules are in Beta</strong>: Starting from version <a href="https://github.com/DataDog/dd-trace-java/releases/tag/v1.34.0">1.34.0</a>, for Java applications, set by-service and by-resource sampling rates from the <a href="/tracing/trace_pipeline/ingestion_controls#configure-the-service-ingestion-rate">Ingestion Control Page</a> UI. Request access to the feature via this <a href="https://forms.gle/WCG57yTCG27BCBB67">link</a>.</div>

> Read more about how to remotely configure sampling rates by service and resource in this [article][1].

Reviewer: ```suggestion
Read more about how to remotely configure sampling rates by service and resource in the [Resource-based sampling guide][1].
```

> **Note**: The remotely set configuration takes precedence over local configuration.

Reviewer: ```suggestion
**Note**: Remotely set configuration takes precedence over local configuration.
```
